.github/workflows/testing.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   test:
     uses: stylelint/.github/.github/workflows/test.yml@main
     with:
-      node-version: '["16", "18", "20"]'
+      node-version: '["18", "20"]'
```

Comment:
I didn't realise that it should have been `'["lts/Hydrogen", "20"]'`.
It's not a big deal because it also has its CON.

i.e. either test 18 instead of 18.12 or test latest Hydrogen LTS instead of the first